README.md

```diff
@@ -344,6 +344,28 @@ These settings can be changed at any time.
 * `settings.updates.version-lock`: Controls the version that will be selected when you issue an update request.  Can be locked to a specific version like `v1.0.0`, or `latest` to take the latest available version.  Defaults to `latest`.
 * `settings.updates.ignore-waves`: Updates are rolled out in waves to reduce the impact of issues.  For testing purposes, you can set this to `true` to ignore those waves and update immediately.
 
+#### Network settings
+
+These settings will impact the networking behavior of the following services:
+* For all variants:
+    * [containerd.service](packages/containerd/containerd.service)
+    * [host-containerd.service](packages/host-ctr/host-containerd.service)
+* For Kubernetes variants:
+    * [kubelet.service](packages/kubernetes-1.18/kubelet.service)
+* For ECS variants:
+    * [docker.service](packages/docker-engine/docker.service)
+    * [ecs.service](packages/ecs-agent/ecs.service)
+
+* `settings.network.http-proxy`: The HTTP proxy server to be used by services listed above.
+* `settings.network.https-proxy`: The HTTPS proxy server to be used by services listed above.
+* `settings.network.no-proxy`: A list of hosts that are excluded from proxying.
+By default, this is set to `["localhost", "127.0.0.1", "169.254.169.254" (Instance Metadata Service endpoint)]`.
```

Comment:
I think having these in `defaults.toml` might be overly complicating things, actually.  Could there be a case where someone wants to proxy these?  (We shouldn't encourage proxying IMDS, and proxying localhost seems... breaky.)

Can we just put these in the EnvironmentFile?  Then the user doesn't have to repeat all these to set their own no-proxy values, which would be a bad experience.  And we can simplify the README here a bit.  (And the handlebars logic in proxy-env, similar to my suggestion in ecs.config.)